src/components/Search/Search.js

```diff
@@ -28,12 +33,18 @@ const SearchBase = props => {
         query: searchString
       })
       .then(res => {
-        props.storeBooks(res.hits);
+        props.storeBooks(withDistance(res.hits, props.coords));
       });
 
     props.history.push(ROUTES.BOOKS);
   };
 
+  if (_.isEmpty(props.books) && props.hasSearched === false) {
+    navigator.geolocation.getCurrentPosition(pos => {
+      props.storeCoords({lat: pos.coords.latitude, lng: pos.coords.longitude});
+    });
+  }
+
```

Comment:
Do you think that we need to call user location once on the beginning and store it on redux?

Reply:
oh, you already store it right, so I will call the redux state on add book page

Reply:
Done!